src/extension.ts

```diff
@@ -65,11 +68,22 @@ function activateTOC(
   // Add the ToC widget to the application restorer:
   restorer.add(toc, 'juputerlab-toc');
 
+  // Attempt to load plugin settings:
+  let settings: ISettingRegistry.ISettings | undefined;
+  try {
+    settings = await settingRegistry.load('@jupyterlab/toc:plugin');
```

Comment:
According to `L148`, the plugin id is `jupyterlab-toc`. Does the argument to `load` need to match that `id`?

Reply:
I got that id from the `package.json` on the first line - `"name": "@jupyterlab/toc",`. I wasn't sure if that'd be the right id to use but it seems to work.

Reply:
Okay. Maybe we should update the plugin `id` on `L148` to match. This would follow, e.g., the Git extension.

Reply:
Ok I see a couple places in the project that reference it. I just added a commit to change the extension id to toc.